applications/gstreamer/gst_video_recorder/python/gst_video_recorder.py

```
@@ -0,0 +1,396 @@
+
+def main() -> int:
+    parser = build_arg_parser()
+    args = parser.parse_args()
+    validate_args(args)
+
+    try:
+        app = GstVideoRecorderApp(args)
+        app.run()
+    except KeyboardInterrupt:
+        return 130
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)